loader/loader.txt

```diff
@@ -0,0 +1,7 @@
+alert("file to load " + file)
+
+function load(file)
+{
+    //loader code here
```

Comment:
Linting, do you guys have preferences to spacing / tabs! For me tab should represent 2 spaces :D